{"description":"These functions provide a mechanism for handling unusual conditions,\nincluding errors and warnings.\n","example_call":"tryCatch(expr, ..., finally)"}